phc/easy/query/api_paging.py

```diff
@@ -45,7 +45,7 @@ def recursive_paging_api_call(
         max_pages = 1
 
     # Compute count and add to progress
-    if _count is None and len(_prev_results) == 0:
+    if _count is None and len(_prev_results) == 0 and False:
```

Comment:
Vestige from testing?

Reply:
Ouch. Yes. That's when I filed the big report about the mismatched count. I'll fix.